generator/model.go

```diff
@@ -804,6 +804,11 @@ func (sg *schemaGenContext) buildProperties() error {
 			return err
 		}
 
+		// set custom serializer tag
```

Comment:
I fail to see why the statement doing the same thing at the end of the loop is not enough. My hunch is that only the type resolver is failing and that piece wouldn't be necessary if the extension was properly resolved for the property (as you have intended to do in types.go)